src/library_promise.js

```diff
@@ -216,7 +216,15 @@ mergeInto(LibraryManager.library, {
     return id;
   },
 
-  emscripten_promise_any__deps: ['$promiseMap', '$idsToPromises'],
+
+  emscripten_promise_any__deps: [
+    '$promiseMap', '$idsToPromises',
+#if !SUPPORTS_PROMISE_ANY
+    // Adding a function here as a dependency will trigger it to be run
+    // at build time, if and only if this function gets included.
```

Comment:
I guess not.  I find this particular method of adding an error to be fairly obtuse though, so for me this can act as a helpful reminder of what is going on here.  Happy to remove it though.